### DISCUSSION TITLE
Fresh 2.1.4 install - SMTP-IP credentials not Authenticating

### DISCUSSION THREAD

**Discussion by intelli-dg** [postalserver/postal#2470] [Help with using Postal]
Everything else is working with SMTP credentials, but I cannot seem to get SMTP-IP credentials to work.  It still gives a 530 Authentication required.

Any guidance?

**Comment by sinnbeck** [comment]
https://github.com/postalserver/postal/issues/2444

**Reply by intelli-dg** [reply]
Thank you so much @sinnbeck !
